docs/src/css/custom.css

```diff
@@ -15,10 +15,15 @@
   --ifm-color-primary-lighter: #0280e3;
   --ifm-color-primary-lightest: #0690fc;
   --ifm-code-font-size: 95%;
+
+  --docusaurus-highlighted-code-line-bg: rgb(0 0 0 / 8%);
+}
+
+[data-theme="dark"] {
+  --docusaurus-highlighted-code-line-bg: rgb(255 255 255 / 8%);
 }
 
 .docusaurus-highlight-code-line {
-  background-color: rgb(72, 77, 91);
```

Comment:
This is a good change, especially for the light theme. Thanks for fixing. For others' reference, here is old vs. new highlights on the light theme:
![image](https://github.com/zmkfirmware/zmk/assets/7876996/e3ac7ba9-f912-40c2-b32d-d71b58540686)